### DISCUSSION TITLE
SP O365 document "partly" index (RefinableString)

### DISCUSSION THREAD

**Discussion by TiloGit** [microsoft-search/pnp-modern-search#3070] [Q&A] [answered]
Dear Community, 

we have some major issues with the SP O365 search index. 
To boil it down we created
new Site (with Site Columns), New Library, added few documents
As you can see in the screenshot some documents have certain Refinable String indexed some not (documents are over 48hrs old and we try reindex as well). 
And yes the documents have a value filed out. 

![sshot-2023-06-12- 12-49-13](https://github.com/microsoft-search/pnp-modern-search/assets/6953606/6eb47213-d1fa-46e9-bba3-038228f24a30)

We have a MS support case open for weeks (L2) but it's doesn't go anywhere. 

Anybody has an suggestion how to fix this or how to speed up the MS support resolution?

**Comment by wobba** [comment]
See https://www.techmikael.com/2020/03/side-effect-of-mapping-crawled.html and by mapping the cp's you likely have removed the default behavior of having the data full text searchable. So a config error, not a product issue.

If the issue is that some items don't have the mapping to refinablestring, that is indeed a support and crawler issue. Did you map at the site level or the tenant level?

**Reply by afnepipe** [reply]
Hi @wobba.
The map is on Site Level.

**Reply by wobba** [reply]
Don't hit reindex many times, as that has an adverse effect on indexing. Requeing over and over. But if some items have a value and others not, that seems like a support issue to log on indexing.

I would have checked the crawl log using e.g. PowerShell to start with and see if that yields info.

**Reply by afnepipe** [reply]
Yes, we already did.
The crawling log shows about 8000 files processed. And out of the 8000, 7990 are successful.
In the logs, I can see that different Craw times.
- 5/25/2023: 1256 crawled documents.
- 5/7/2023: 2740 crawled documents
- 5/7/2023: 2993 crawled documents.
We are currently working with a SharePoint library containing over 7000 documents, and with the information described above we can validate that many documents were omitted during the crwaling run.

**Reply by wobba** [reply]
Do you have a crawl log entry for the failed items? Output with raw format.

**Reply by afnepipe** [reply]
![image](https://github.com/microsoft-search/pnp-modern-search/assets/96144314/0d6d9908-50e4-426e-9821-a024a2453596)
I found in the log level: Success and Warnings 

I don't if I answer your question. Please let me know.

**Reply by afnepipe** [reply]
Mikael, 
To confirm what you sent me earlier "See https://www.techmikael.com/2020/03/side-effect-of-mapping-crawled.html and when mapping the cp's you probably removed the default behavior of having the data full text searchable. So a configuration error, not a product issue."

First of all, what do the cp's mean? I am confused.
Could you share with me an example/screenshot of a mapping configuration? I don't know if I removed the full text data by mistake.

**Reply by wobba** [reply]
Use -RawFormat when running the crawllog cmdlet as it gives more detail into the warnings/errors.

**Reply by afnepipe** [reply, marked as answer]
Hi @wobba.
The issue was resolved.
For unknown reasons properties mapped as RefinableString were marked as "Excluded" on the searchable columns page.

This page is normally hidden from the site owner. Therefore, we do not know exactly why the properties were marked.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/96144314/97f6db0b-6f07-4562-b9ef-f3d6753f8b8c)

You can mark this discussion as resolved

**Reply by wobba** [reply]
Happy you found the root cause :)
